Cubical/Algebra/CommAlgebra/QuotientAlgebra.agda

```diff
@@ -186,21 +207,22 @@ module _ {R : CommRing ℓ} (A : CommAlgebra R ℓ) (I : IdealsIn A) where
                             (isPropIsCommIdeal (CommAlgebra→CommRing A))
                             refl ⟩
     _                  ≡⟨  CommRing.kernel≡I {R = CommAlgebra→CommRing A} I ⟩
-    I ∎
+    I                  ∎
 
 
 private
-  module _ (R : CommRing ℓ) where
+  module _ (R : CommRing ℓ) where abstract
     open CommRingStr (snd R)
     lemma : (y : (fst R)) → y ≡ y - 0r
     lemma = solve R
 
-isZeroFromIdeal : {R : CommRing ℓ} {A : CommAlgebra R ℓ} {I : IdealsIn A}
-                  → (x : ⟨ A ⟩) → x ∈ (fst I) → fst (quotientHom A I) x ≡ CommAlgebraStr.0a (snd (A / I))
-isZeroFromIdeal {A = A} {I = I} x x∈I = eq/ x 0a (subst (λ y → y ∈ (fst I)) step x∈I )
-  where
-    open CommAlgebraStr (snd A)
-    step : x ≡ x - 0a
-    step = lemma (CommAlgebra→CommRing A) x
-    0' : ⟨ A / I ⟩
-    0' = fst (quotientHom A I) 0a
+abstract
```

Comment:
Merely a cosmetic suggestion: When reading, it is a bit surprising to see just an 'abstract' all of a sudden - maybe you could write it in the same style as above, i.e. in an annoynmous module?

Reply:
Good idea, I will do that.